include/arch/arm/arch/64/mode/object/structures.bf

```diff
@@ -271,34 +241,25 @@ tagged_union asid_map type {
 -- PGDE, PUDE, PDEs and PTEs, assuming 48-bit physical address
 base 64(48,0)
 
-block pgde_invalid {
-    padding                         62
-    field pgde_type                 2
-}
 
-block pgde_pud {
-    padding                         16
-    field_high pud_base_address     36
+block pte_table {
+    padding                         5
+    field pte_sw_type               1
```

Comment:
There is an explanation about this on the definition of  `pte_type`:
There is two page type fields because the 4k page size uses a different hardware encoding. We use bit 58 which is reserved for software use to encode this difference in the tag for these types.